modules/installation-azure-limits.adoc

```diff
@@ -96,7 +119,9 @@ security groups for the control plane and for the compute node subnets:
 
 |Network load balancers
 | 3
```

Comment:
UPI we are only really creating 2 load balancers. API & Ingress

Reply:
@patrickdillon the description for *Network load balancers* is the following:

>Each cluster creates the following [load balancers](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview):
>
>default: Public IP address that load balances requests to ports 80 and 443 across worker machines
>
>internal: Private IP address that load balances requests to ports 6443 and 22623 across control plane machines
>
>external: Public IP address that load balances requests to port 6443 across control plane machines
>
>If your applications create more Kubernetes `LoadBalancer` service objects, your cluster uses more load balancers.

Do you propose we remove the `default` load balancer type? And you said for UPI, so this applies for both Azure and ASH?

Reply:
No, combine internal and external into one, you can call it API.